holoviews/element/comparison.py

```diff
@@ -648,3 +656,140 @@ def __init__(self, *args, **kwargs):
         registry = Comparison.register()
         for k, v in registry.items():
             self.addTypeEqualityFunc(k, v)
+
+
+
+
+############################################################
+# Archive comparison
+
+ComparisonResult = namedtuple('ComparisonResult', ['ref_archive', 'test_archive', 'ref_member', 'match', 'msg'])
+
```

Comment:
Just a minor comment, I would use the names: `` ['reference', 'test', 'file', 'match', 'msg']``